nengo/builder/ensemble.py

```diff
@@ -49,6 +49,23 @@ def get_activities(model, ens, eval_points):
         x, model.params[ens].gain, model.params[ens].bias)
 
 
+def get_gain_bias(ens, rng=np.random):
```

Comment:
According to the OP,
> (so that I can reuse that code in `nengo_lasagne`)

As long as there's a unit test for this new function, and for the new use case in general, I'm happy with the rest.